tests/legacy.c

```diff
@@ -133,8 +133,8 @@ static int testStreamingAPI(void)
 
 static int testFrameDecoding(void)
 {
-    if (ZSTD_decompressBound(COMPRESSED, COMPRESSED_SIZE) != ZSTD_CONTENTSIZE_ERROR) {
-        DISPLAY("ERROR: ZSTD_decompressBound: Expected to receive ZSTD_CONTENTSIZE_ERROR\n");
+    if (strlen(EXPECTED) > ZSTD_decompressBound(COMPRESSED, COMPRESSED_SIZE)) {
```

Comment:
Can you also add a test for `ZSTD_findFrameCompressedSize()`, since we are modifying that code? This data contains multiple frames, since we should make sure if we sum them all up we get the right size.

Reply:
Definitely. I added a test for `ZSTD_findFrameCompressedSize` in the latest commit. Let me know if that's what you were thinking.